src/blacksmith/domain/model/params.py

```diff
@@ -392,6 +392,20 @@ def unwrap_or_else(self, op: Callable[[TError_co], TResponse]) -> TResponse:
         """
         return self._result.unwrap_or_else(op)
 
+    def unwrap_or_raise(self, exc: Type[Exception]) -> TResponse:
+        """
+        Return the response or raise the exception exc.
+
+        :raises exc: it the response is an error.
+        :raises NoResponseSchemaException: if there are no response schema set.
+        """
+        # not that we can't use the unwrap_or_raise from the result
+        # because/until we support version that don't have the method
+        # for python 3.7 compatibility.
```

Comment:
```suggestion
        :raises exc: if the response is an error.
        :raises NoResponseSchemaException: if there are no response schema set.
        """
        # note that we can't use the unwrap_or_raise from the result
        # because/until we support version that don't have the method
        # for python 3.7 compatibility.
```